src/BME280I2C.cpp

```diff
@@ -40,6 +40,7 @@ BME280I2C::BME280I2C
 ):BME280(settings),
   m_settings(settings)
 {
+  Wire.begin();
```

Comment:
This is on the user, per previous issue

Reply:
My investigations showed that `Wire.begin()` is necessary to have a non blocking behaviour for `Wire.endTransmission()`.
I just tested it here again.